### PR TITLE
Update service-quick-create-sharepoint-list.md

### PR DESCRIPTION
**Pull request by amandariveraMSFT**
Added a new consideration for the feature. 

CC @maggiesMSFT

### CODE REVIEW

**Comment by PRMerger9** [comment]
@amandariveraMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ShannonLeavitt** [comment]
@maggiesMSFT - Can you review the proposed changes? IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

Note: Before we merge, we can fix the typo (ment -> meant).

#label:"aq-pr-triaged"
#assign:@MicrosoftDocs/public-repo-pr-review-team

**Comment by PRMerger19** [comment]
The following users don't have valid GitHub IDs, or are not collaborators on this repo: microsoftdocs/public-repo-pr-review-team

File: powerbi-docs/create-reports/service-quick-create-sharepoint-list.md [hunk 1]
```diff
@@ -84,6 +84,7 @@ Anyone can use the **Visualize the list** or **visualize the library** feature t
 - Very large lists may be slow or may time out altogether. 
 - This feature isn't supported for guest users in a tenant.
 - Within SharePoint libraries, this option will only show on the root of the document library.
+- The reports and datasets created through this SharePoint experience are stored in special system generated workspaces hosted on shared capacity. Users are not ment to access these workspaces outside of the SharePoint experience.
```
Comment (maggiesMSFT) [inline powerbi-docs/create-reports/service-quick-create-sharepoint-list.md:87]:
```suggestion
- The reports and datasets created through this SharePoint experience are stored in special, system-generated workspaces hosted in shared capacity. Users aren't meant to access these workspaces outside of the SharePoint experience.
```

**Review by maggiesMSFT — APPROVED** [review]

**Comment by maggiesMSFT** [comment]
#sign-off

**Pull request merged**